docs/pages/blog/introducing-pigment-css.md

```diff
@@ -78,19 +85,53 @@ Check out [How Pigment CSS works](https://github.com/mui/pigment-css/blob/maste
 
 For users of Emotion and styled-components, the benefits of adopting Pigment CSS are clear: your end users get better performance, and you get RSC and App Router compatibility without having to significantly change how you author component styles.
 
-### Better performance
+### Better page load performance
+
+When comparing the same Material UI app built with Next.js and either Emotion or Pigment CSS, we've observed the following page load performance gains using the same code, but powered by different style engines:
+
+<!-- vale MUI.CorrectReferenceAllCases = NO -->
+
+| Metrics             |                                                                                                        Emotion |                                                                                                        Pigment CSS | Change |
+| :------------------ | -------------------------------------------------------------------------------------------------------------: | -----------------------------------------------------------------------------------------------------------------: | -----: |
+| First Load JS       |                                                            [131 kB](https://emotion-demo.vercel.app/?metric=1) |                                                            [104 kB](https://pigment-css-demo.vercel.app/?metric=1) |   -20% |
+| Total blocking time | [280 ms](https://pagespeed.web.dev/analysis/https-emotion-demo-vercel-app/uxfpkvbp31?hl=fr&form_factor=mobile) | [210 ms](https://pagespeed.web.dev/analysis/https-pigment-css-demo-vercel-app/azcw1qxkec?hl=fr&form_factor=mobile) |   -25% |
+
+<!-- vale MUI.CorrectReferenceAllCases = YES -->
+
+:::info
+**Reproduction**. Because you should only trust numbers that you can reproduce, we've shared below how we found these numbers.
```

Comment:
```suggestion
To learn more about how we got to these numbers and how to reproduce them, check out the live demo and repositories with Emotion and Pigment CSS.
```
We don't need to say what visitors should do. I'd go with something like this.

Reply:
Agree 👍